.github/workflows/build.yml

```diff
@@ -55,6 +55,11 @@ jobs:
         with:
           cli: ${{ env.CLOJURE_VERSION }}
 
+      - uses: actions/checkout@v2
```

Comment:
This is mistakenly introduced. will fix it.

Reply:
My bad. Thanks for fixing!